sql/moz-fx-data-shared-prod/mozilla_vpn_derived/waitlist_funnel_v1/query.sql

```diff
@@ -0,0 +1,102 @@
+WITH send_ids AS (
+  SELECT
+    SendID,
+    ARRAY_AGG(DISTINCT URL) AS URLs,
+  FROM
+    `mozilla-cdp-prod`.sfmc.clicks
+  WHERE
+    URL LIKE "%://vpn.mozilla.org%"
+  GROUP BY
+    SendID
```

Comment:
I discussed with marketing and determined that links to `vpn.mozilla.org` are the best available way to detect `SendID`s for mozilla vpn. Unfortunately a definitive call-to-action URL isn't available in `sfmc.sendjobs`, so URLs that match the pattern are preserved and used later when joining on `sfmc.clicks`, which contains clicks for other URLs in the email, such as to the privacy policy.